src/public/video.ts

```diff
@@ -66,6 +78,25 @@ export namespace video {
     EffectDisabled,
   }
 
+  export interface PersonalizedEffect {
+    /**
+     * personalized effect id
+     */
+    id: string;
+    /**
+     * display name
+     */
+    name: string;
+    /**
+     * effect type defined by app
+     */
+    type: string;
+    /**
+     * URL of the thumbnail
```

Comment:
How are these URLs used on the Teams side? Are the protected by any kind of authentication?